src/api/VotingIrregularities.Domain/NotaAggregate/AdaugaNotaCommand.cs

```diff
@@ -17,17 +17,17 @@ public class AdaugaNotaCommand : IRequest<int>
 
     public class NotaProfile : Profile
     {
-        public NotaProfile()
-        {
-            CreateMap<AdaugaNotaCommand, Note>()
-                .ForMember(dest => dest.IdQuestion, c => c.MapFrom(src =>
-                    !src.IdIntrebare.HasValue || src.IdIntrebare.Value <= 0 ? null : src.IdIntrebare)
-                )
-                .ForMember(dest => dest.LastModified, c => c.MapFrom(src => DateTime.UtcNow))
-                .ForMember(dest => dest.AttachementPath, c => c.MapFrom(src => src.CaleFisierAtasat))
-                .ForMember(dest => dest.IdObserver, c => c.MapFrom(src => src.IdObservator))
-                .ForMember(dest => dest.IdPollingStation, c => c.MapFrom(src => src.IdSectieDeVotare))
-                .ForMember(dest => dest.Text, c => c.MapFrom(src => src.TextNota));
-        }
+        //public NotaProfile()
```

Comment:
Do the old endpoints(`api/v1/note`) still work if you comment this section?

Reply:
The old endpoint was rendered useless by previous pushes.... Also, marking stuff with obsolete/deprecated flags shouldn't be done on said things that are supposed to be working/backwards compatible. A comment is more than enough.

Anyways, I've uncommented the said part, restored the 'AdaugaNotaHandler' and tested that the endpoint is actually working as it was supposed to.